.github/workflows/stats.yml

```diff
@@ -56,6 +56,11 @@ jobs:
         run: cargo test --locked --workspace --all-features --doc
         if: success() || failure()
 
+      - name: ENVs in documentation
+        run: cargo run --bin check-envs
```

Comment:
you changed to env-docs-generation, so either change it to `check-envs` for consistency, or change this step to 

`cargo run --bin env-docs-generation`

Reply:
oops, missed it